src/components/setup/pages/WelcomePage.tsx

```diff
@@ -22,6 +22,8 @@ const WelcomePage: FunctionComponent<SetupPageProps> = ({nextPage}) => {
                 break;
             case "syncthing-gtk":
                 service_or_flatpak_name = "me.kozec.syncthingtk"
+            case "syncthing-tray":
+                service_or_flatpak_name = "io.github.martchus.syncthingtray"
```

Comment:
This doesn't work, since the case above has no break it would now also set it to use syncthingtray if I select Syncthing GTK. Either this logic must be changed to not be a switch with fallthroughs anymore or you could work with the null coalleasing operator and only set the value if it wasn't set before.